### PR TITLE
nginx configs for local url

### PR DESCRIPTION
**Pull request by shtukas**
Introduces the nginx config file to use the local url `https://explainer.local.dev-gutools.co.uk/`

TO TEST: 
- checkout the branch
- run according to the readme instructions 
- go to https://explainer.local.dev-gutools.co.uk/

### CODE REVIEW

**Comment by shtukas** [comment]
tested on local :)

File: README.md [hunk 1]
```diff
@@ -19,11 +19,18 @@ $ cat /etc/gu/explainer.stage.conf
 stage=DEV
 ```
 
+Configure the composer route in nginx:
+
+```
+cd <path_of_dev_nginx>
+sudo ./setup-app.rb <path_of_explainer>/nginx/nginx-mapping.yml
```
Comment (jamesgorrie) [inline README.md:26]: As a point, you should be able to run:
`<path_of_dev-nginx>/setup-app.rb ./nginx/nginx-mapping.yml`

Making it more project specific.

**Comment by jamesgorrie** [comment]
👍

File: README.md (hunk 1 above)
Comment (shtukas) [inline README.md:26]: Yes :)
But for some reason many of the other Editorial Tools READMEs run that command from within the nginx folder. I just let myself contaminated by the existing conventions :)

**Pull request merged**